backend/tilavarauspalvelu/models/reservation_unit/queryset.py

```diff
@@ -46,6 +45,10 @@ def with_first_reservable_time(
         cache_key: str = "",
     ) -> Self:
         """Annotate the queryset with `first_reservable_time` and `is_closed` for each reservation unit."""
+        from tilavarauspalvelu.services.first_reservable_time.first_reservable_time_helper import (
+            FirstReservableTimeHelper,
+        )
+
```

Comment:
Why does this need to be inline? QuerySets should be lazy now, so there shouldn't be import loops?

Reply:
Does not need to be inline anymore. Forgot to fix this after your PR was merged :)